clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -845,6 +882,30 @@ private void handleListOffsetResponse(Map<TopicPartition, Long> timestampsToSear
         return parsedRecords;
     }
 
+    private PriorityQueue<FetchResponse.AbortedTransaction> getAbortedTransactions(FetchResponse.PartitionData partition) {
```

Comment:
We can let this function to only return `Queue` then in the `else` condition we can return an empty abstract queue.